src/server/snap.rs

```diff
@@ -488,6 +488,7 @@ impl<R: RaftExtension + 'static> Runnable for Runner<R> {
                 let recving_count = self.recving_count.clone();
```

Comment:
I think it would be better to have SNAP_TASK_COUNTER_STATIC.recv2.inc() for easier troubleshooting.